client/components/Events/ManageEventTemplatesModal.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/no-multi-comp */
+
+import {IFormGroup, IBaseRestApiResponse, IGenericListPageComponent} from 'superdesk-api';
+import {superdeskApi} from '../../superdeskApi';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Modal} from '../index';
+
+interface IProps {
+    handleHide(): void;
+}
+
+interface IEventTemplate extends IBaseRestApiResponse {
+    template_name: string;
+}
+
+export class ManageEventTemplatesModal extends React.PureComponent<IProps> {
```

Comment:
This component does not cater to notifications to update the list items. When templates are removed/edited, the list should update accordingly. Please refer to agenda management modal behavior.